src/coreclr/src/jit/compiler.h

```diff
@@ -3761,7 +3761,8 @@ class Compiler
 
     GenTree* getArgForHWIntrinsic(var_types argType, CORINFO_CLASS_HANDLE argClass, bool expectAddr = false);
     GenTree* impNonConstFallback(NamedIntrinsic intrinsic, var_types simdType, var_types baseType);
-    GenTree* addRangeCheckIfNeeded(NamedIntrinsic intrinsic, GenTree* lastOp, bool mustExpand, int immUpperBound);
+    GenTree* addRangeCheckIfNeeded(
+        NamedIntrinsic intrinsic, GenTree* lastOp, bool mustExpand, int immLowerBoundIncl, int immUpperBoundExcl);
```

Comment:
This should be `Incl` now

Reply:
I will just remove the suffixes for both since now both - the upper and lower bounds are inclusive.